bundle/phases/initialize.go

```diff
@@ -26,36 +26,70 @@ func Initialize(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 	log.Info(ctx, "Phase: initialize")
 
 	return bundle.ApplySeq(ctx, b,
+		// Reads (dynamic): resource.*.*
+		// Checks that none of resources.<type>.<key> is nil. Raises error otherwise.
 		validate.AllResourcesHaveValues(),
+
+		// Reads (dynamic): workspace.{host,profile,...} (ensure that there are no variable references)
 		validate.NoInterpolationInAuthConfig(),
 
-		// Update all path fields in the sync block to be relative to the bundle root path.
+		// Updates (dynamic): sync.{paths,include,exclude} (makes them relative to bundle root rather than to definition file)
+		// Rewrites sync paths to be relative to the bundle root instead of the file they were defined in.
 		mutator.RewriteSyncPaths(),
 
+		// Reads (dynamic): sync.paths (checks that it is absent)
+		// Updates (typed): b.Config.Sync.Paths (default set to ["."])
 		// Configure the default sync path to equal the bundle root if not explicitly configured.
 		// By default, this means all files in the bundle root directory are synchronized.
 		mutator.SyncDefaultPath(),
 
 		// Figure out if the sync root path is identical or an ancestor of the bundle root path.
 		// If it is an ancestor, this updates all paths to be relative to the sync root path.
+		// Reads (typed): b.Config.Sync.Paths (calculates longest common parent together with bundle root).
+		// Updates (typed) b.{SyncRoot,SyncRootPath}  (set to calculate sync root, which is either bundle root or some parent of bundle root)
+		// Updates (typed) b.Config.{Sync,Include,Exclude} they set to be relative to SyncRootPath instead of bundle root
 		mutator.SyncInferRoot(),
 
+		// Reads (typed): b.Config.Workspace.CurrentUser (checks if it's already set)
+		// Updates (typed): b.Config.Workspace.CurrentUser (sets user information from API)
+		// Updates (typed): b.Tagging (configures tagging object based on workspace client)
```

Comment:
This comment does not actually explain what do we do there, I believe we set a proper tagginh object based on the cloud we run at

Reply:
thanks, fixed